<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_controlClass</class>
 <widget class="QMainWindow" name="main_controlClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main_control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>391</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei</family>
        <pointsize>36</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>井字棋游戏</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开始游戏</string>
      </property>
      <property name="icon">
       <iconset resource="main_control.qrc">
        <normaloff>:/resources/u=2361293748,1681636035&amp;fm=26&amp;fmt=auto.webp</normaloff>:/resources/u=2361293748,1681636035&amp;fm=26&amp;fmt=auto.webp</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>151</width>
        <height>61</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>330</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出游戏</string>
      </property>
      <property name="icon">
       <iconset resource="main_control.qrc">
        <normaloff>:/resources/u=2361293748,1681636035&amp;fm=26&amp;fmt=auto.webp</normaloff>:/resources/u=2361293748,1681636035&amp;fm=26&amp;fmt=auto.webp</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>221</width>
        <height>61</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
    <widget class="QWidget" name="page_3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionnew"/>
    <addaction name="actionopen"/>
    <addaction name="separator"/>
    <addaction name="actionquit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionreturn"/>
    <addaction name="separator"/>
    <addaction name="actioncopy"/>
    <addaction name="actionstick"/>
   </widget>
   <widget class="QMenu" name="menuGIT">
    <property name="title">
     <string>GIT</string>
    </property>
    <addaction name="actionGIT"/>
    <addaction name="actionPULL"/>
    <addaction name="actionPUSH"/>
    <addaction name="actionCLONE"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionhelp"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menuGIT"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionnew"/>
   <addaction name="actionopen"/>
   <addaction name="separator"/>
   <addaction name="actioncopy"/>
   <addaction name="actionstick"/>
   <addaction name="separator"/>
   <addaction name="actionGIT"/>
   <addaction name="actionPULL"/>
   <addaction name="actionPUSH"/>
   <addaction name="actionCLONE"/>
   <addaction name="separator"/>
   <addaction name="actionhelp"/>
   <addaction name="actionabout"/>
   <addaction name="separator"/>
   <addaction name="actionquit"/>
  </widget>
  <action name="actionnew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionquit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionreturn">
   <property name="text">
    <string>测回</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionstick">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionGIT">
   <property name="text">
    <string>GIT</string>
   </property>
  </action>
  <action name="actionPULL">
   <property name="text">
    <string>PULL</string>
   </property>
  </action>
  <action name="actionPUSH">
   <property name="text">
    <string>PUSH</string>
   </property>
  </action>
  <action name="actionCLONE">
   <property name="text">
    <string>CLONE</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="text">
    <string>help</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="main_control.qrc"/>
 </resources>
 <connections/>
</ui>
